### ISSUE TITLE
[Android with self-hosted tiles] using mapview requires calling mapbox.getinstance

### ISSUE THREAD

**Issue by bahung1221** [react-native-mapbox-gl/maps#1134]
Hi

**Describe the bug**  

I'm using `react-native-mapbox-gl/maps` to render my map without need any tiles or styles from Mapbox service. All tiles, styles, fonts were self-hosted on my server. 
So I didn't have any Mapbox access token, it worked very well on iOS, but doesn't work on android. 

I found a PR from [mapbox/mapbox-gl-native](https://github.com/mapbox/mapbox-gl-native/pull/12001) that was merged on 2018, it seems that PR allowed us to omit the access token. 
So I think `react-native-mapbox-gl/maps` should allow us to omit the access token, too.

**To Reproduce**  
I don't have a reproduce now, because my source code is meshed up and also it's company resource 😢 If you still need a reproduce, I will try to create a sample repo that uses OSM tiles instead of Mapbox tiles.

Please include a single standalone React Native component.  
```
<Container>
    <MapboxGL.MapView
        ref={this.setMapRef}
        styleURL={offlineStyleUri}
        style={styles.map}
        onRegionDidChange={this.onRegionDidChange}
        logoEnabled={false}
        attributionEnabled={false}
    >
        ...
    </MapboxGL.MapView>
</Container>
```

style is JSON file like this one (but with my own server): https://raw.githubusercontent.com/maputnik/osm-liberty/gh-pages/style.json

**Expected behavior**  
Able to use `react-native-mapbox-gl/maps` without access token when used self-hosted server for both Android and iOS.

**Actual behavior**  
Only work on iOS, Android was blocked.

**Screenshots**  
<img width="301" alt="Screen Shot 2020-12-26 at 18 20 13" src="https://user-images.githubusercontent.com/27690861/103150774-c1b76980-47a9-11eb-9955-29c5f8fabcfa.png">

**Versions (please complete the following information):**  
 - Platform: Android
 - Platform OS: Android 7
 - Device: Nexus 5
 - Emulator/ Simulator: Yes
 - Dev OS: macOs 11.1
 - react-native-mapbox-gl: 8.0.0
 - React Native Version: 0.63.2

**Comment by bahung1221** [comment]
I found that `mapbox/mapbox-gl-native` is dead, Mapbox Android SDK was moved to [mapbox/mapbox-gl-native-android](https://github.com/mapbox/mapbox-gl-native-android) instead.

I check their code and it seems that this issue comes from their side, not `react-native-mapbox-gl/maps`. I opened an issue here: https://github.com/mapbox/mapbox-gl-native-android/issues/632.

So I go to close this issue, thanks!
